sources/packages/web/src/components/common/AppealRequestsForm.vue

```diff
@@ -18,15 +18,15 @@
 </template>
 <script lang="ts">
 import { AESTRoutesConst } from "@/constants/routes/RouteConstants";
-import { defineComponent } from "vue";
+import { defineComponent, PropType } from "vue";
 import { useFormioUtils } from "@/composables";
 import { StudentAppealRequest } from "@/types";
 
 export default defineComponent({
   emits: ["submitted"],
   props: {
     studentAppealRequests: {
-      type: Object,
+      type: Object as PropType<StudentAppealRequest>,
```

Comment:
Thanks for adding the PropType but the property defined as Object would cause a warning.
The recommended way that we adjusted in the past should be.
```
type: String as PropType<StudentAppealRequest>,
```

Reply:
Comment wrongly added.